ironfish/src/wallet/exporter/encoders/json.ts

```diff
@@ -66,6 +66,7 @@ type AccountEncodedJSON = {
     publicKeyPackage: string
   }
   proofAuthorizingKey?: string | null
+  networkId?: number
```

Comment:
I think accounts are pretty translatable across networks, but I was kind of imagining that the networkId is more a function of the createdAt information. I think this is fine, but you might wonder why an exported account has a networkId. Does it not work across networks? Maybe they don't so this is fine.

In my head I was imaginging
```ts
type AccountImport = {
    createdAt?: {
        networkId?: number
        sequence?: number
    }
}
```

Reply:
I think it makes sense to do it this way -- I agree that the networkId is only relevant to the information in createdAt.

I'll update to nest it in that field.